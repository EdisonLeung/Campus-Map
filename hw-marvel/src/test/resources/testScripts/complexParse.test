# Tests parsing for a complex graph

LoadGraph complexGraph complexGraph.csv

ListNodes complexGraph
ListChildren complexGraph a
ListChildren complexGraph b
ListChildren complexGraph c
ListChildren complexGraph d