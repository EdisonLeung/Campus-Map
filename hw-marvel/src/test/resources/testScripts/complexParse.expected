# Tests parsing for a complex graph

loaded graph complexGraph

complexGraph contains: a b c d
the children of a in complexGraph are: b(ABa) b(ABb) c(AC) d(AD)
the children of b in complexGraph are: a(ABa) a(ABb)
the children of c in complexGraph are: a(AC) d(CD)
the children of d in complexGraph are: a(AD) c(CD)